src/main/java/io/servertap/PluginEntrypoint.java

```diff
@@ -66,14 +66,16 @@ public void onEnable() {
                 get("players", PlayerApi::playersGet);
                 get("players/:player", PlayerApi::playerGet);
                 get("allPlayers", PlayerApi::offlinePlayersGet);
-
+                get("player/:uuid/:world/inventory", PlayerApi::getPlayerInv);
```

Comment:
Can you change this to `players/:uuid/:world/inventory` to be consistent with the other player routes?